src/OpenTelemetry.Api/Context/Propagation/TextFormatContext.cs

```diff
@@ -0,0 +1,110 @@
+﻿// <copyright file="TextFormatContext.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace OpenTelemetry.Context.Propagation
+{
+    /// <summary>
+    /// TextFormat context.
+    /// </summary>
+    public readonly struct TextFormatContext : IEquatable<TextFormatContext>
```

Comment:
This represents both SpanContext and Baggage. Not sure if the name TextFormatContext is aptly capturing the purpose.

Reply:
Shooting down a name without a better suggestion isn't fair 🤣 I went with `PropagationContext`. Probably some further improvements needed. Let's do a follow-up PR? Our `ITextFormat` name doesn't really mirror what the other libraries are doing. All I have checked have HttpTextFormat type of things, which is what is called out in the spec. Some chain like TraceContextHttpPropagationFormat is IHttpPropagationFormat is IPropagationFormat?

Reply:
I like `IPropagationFormat` :)